java/buildconf/build-props.xml

```diff
@@ -108,7 +127,7 @@
       value="bcel ${cglib-jar} commons-beanutils commons-cli commons-codec
              commons-collections commons-digester commons-discovery
              commons-el commons-fileupload commons-io ${commons-lang} commons-logging ${commons-validator} ${hibernate}
-             ${javamail} smtp jdom ${jmock-jars}
+	     ${javamail} smtp jdom ${jmock-jars}
```

Comment:
Seems like a few spaces have mistakenly been replaced by a tab...

Reply:
Thank you, I couldn't figure out what was wrong with it! Didn't think of tabs.